modules/reference/pages/feature/Jakarta-EE9-feature-updates.adoc

```diff
@@ -0,0 +1,220 @@
+//
+:page-description: For Jakarta EE 9 applications to run in Open Liberty, Jakarta EE 9 features must be specified in the server configuration. If Jakarta EE 8 feature name is incorrectly used with the Jakarta EE 9 version, the server does not start, and the server logs provide information to say which new feature name must be used.
+:seo-title: Jakarta EE9 feature updates - OpenLiberty.io
+:projectName: Open Liberty
+:page-layout: Jakarta EE 9
+:page-type: feature updates
+= Jakarta EE 9 feature updates
+
+For Jakarta EE 9 applications to run in Open Liberty, Jakarta EE 9 features must be specified in the server configuration. If Jakarta EE 8 feature name is incorrectly used with Jakarta EE 9 version, the server does not start, and the server logs provide information to say which new feature name must be used.
+
+In many cases, Jakarta EE 9 features use the same short names as the corresponding Jakarta EE 8 features. In those cases, the Jakarta EE 9 features simply have an updated feature version.
+For example, the Contexts and Dependency Injection feature is updated from `cdi-2.0` to `cdi-3.0`.
+
+However, in some cases, the feature short name is also different, as with the Enterprise JavaBeans 3.2 feature `ejb-3.2`, which was updated to Enterprise Beans 4.0 `enterpriseBeans-4.0`.
+For example, if `ejb-3.2` is incorrectly changed to `ejb-4.0`, the server logs indicate that `enterpriseBeans-4.0` must be used.
+
+
+The following table lists features for which only the feature version is updated:
+
+.Jakarta EE 8 to Jakarta EE 9 feature updates: version only
+|===
+| Feature name | Jakarta EE 8 feature name | Jakarta EE 9 feature name
+
+| Application Client Support for Server
+| appClientSupport-1.0
+| appClientSupport-2.0
+
+| Application Security
+| appSecurity-3.0
+| appSecurity-4.0
+
+| Batch API
+| batch-1.0
+| batch-2.0
+
+| Bean Validation
+| beanValidation-2.0
+| beanValidation-3.0
+
+| Contexts and Dependency Injection
+| cdi-2.0
+| cdi-3.0
+
+| Concurrency Utilities
+| concurrent-1.0
+| concurrent-2.0
+
+| Java EE
+| jakartaee-8.0
+| jakartaee-9.1
+
+| JavaScript Object Notation Binding
+| jsonb-1.0
+| jsonb-2.0
+
+| JavaScript Object Notation Binding Container
+| jsonbContainer-1.0
+| jsonbContainer-2.0
+
+| JavaScript Object Notation Processing
+| jsonp-1.1
+| jsonp-2.0
+
+| JavaScript Object Notation Processing Container
+| jsonpContainer-1.1
+| jsonpContainer-2.0
+
+| Java EE Managed Bean
+| managedBeans-1.0
+| managedBeans-2.0
+
+| Message-Driven Beans
+| mdb-3.2
+| mdb-4.0
+
+| Java Servlet
+| servlet-4.0
+| servlet-5.0
+
+| Java EE Web Profile
+| webProfile-8.0
+| webProfile-9.1
+
+| Java WebSocket
+| websocket-1.1
+| websocket-2.0
+|===
+
+
+The following table lists features for which both the feature short name and the feature version are updated:
+
+.Jakarta EE 8 to Jakarta EE 9 feature updates: short name and version
+|===
```

Comment:
In some cases, both the long and short feature names have changed. For example, 

EE8:  Enterprise JavaBeans (ejb-3.2)
EE9:  Enterprise Beans (enterpriseBeans-4.0)

I think we need to document the name long names as well. Rather than a separate column, might just include both values for each in the same column- using parentheses for the shortname. We'll have to confirm the feature names with Jared where they are not obivous (ie just change from Java->Jakarta)